build.gradle.kts

```diff
@@ -135,49 +132,115 @@ subprojects {
         }
     }
 
-    // publish
-    artifactory {
-        setContextUrl("<<//todo maven-repo-url-here>>")
-        publish(
-            delegateClosureOf<PublisherConfig> {
-                repository(
-                    delegateClosureOf<GroovyObject> {
-                        setProperty("repoKey", getArtifactoryRepo(project.version.toString()))
-                        setProperty("username", project.findProperty("artifactory_user") ?: "dummy_user")
-                        setProperty("password", project.findProperty("artifactory_password") ?: "dummy_password")
-                        setProperty("maven", true)
+    if (!project.name.contains("example")) {
+        publishing {
+            publications {
+                create<MavenPublication>("maven") {
+                    from(components["java"])
+                    afterEvaluate {
+                        artifactId = "auditor-".plus(tasks.jar.get().archiveBaseName.get())
+                        group = "io.github.lowes"
+                        version = getAuditorVersion()
                     }
-                )
-                defaults(
-                    delegateClosureOf<GroovyObject> {
-                        invokeMethod("publications", "mavenJava")
+                    pom {
+                        name.set("Auditor")
+                        description.set("Auditing Library for JVM apps")
+                        url.set("https://github.com/lowes/auditor")
+                        licenses {
+                            license {
+                                name.set("Apache-2.0")
+                                url.set("https://opensource.org/licenses/Apache-2.0")
+                            }
+                        }
+                        developers {
+                            developer {
+                                id.set("akhilsunilkumar")
```

Comment:
we should limit the names to the lowe's open source contact and avoid any personal names here.

Reply:
Ok.. I'll Add Just Lowes